### PR TITLE
Format with black

### PR DESCRIPTION
**Pull request by sloria**
Adds pre-commit configuration.

(Optionally) install pre-commit hooks with

```
pre-commit install
```

To run formatting manually:

```
pre-commit run --all-files
```

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+repos:
+- repo: https://github.com/asottile/pyupgrade
```
Comment (sloria) [inline .pre-commit-config.yaml]: pyupgrade isn't strictly necessary here, but it's a nice check to make sure you're using up-to-date syntax.

**Comment by lovasoa** [comment]
Thank you for this PR! I like the idea of standardizing style. I'll merge this once you fix the CI error.

**Comment by sloria** [comment]
OK, will look into the CI error later today.

File: marshmallow_dataclass/__init__.py [hunk 2]
```diff
@@ -35,39 +35,32 @@ class User:
       Schema: ClassVar[Type[Schema]] = Schema # For the type checker
 """
 import dataclasses
-import datetime
```
Comment (sloria) [inline marshmallow_dataclass/__init__.py]: Woops, I missed these in #35. flake8 spotted them.

**Comment by sloria** [comment]
By the way, feel free to use "Squash and Merge" on my PRs. I don't care about having all my commits in the history. GitHub will link to the PR anyway.

**Pull request merged**